### DISCUSSION TITLE
Couldn't find anyway to update slides per view in Swiper v.10

### DISCUSSION THREAD

**Discussion by masarbazi** [nolimits4web/swiper#6890] [Q&A] [answered]
### Clear and concise description of the problem

Hi, I have used Swiper pre versions and I used ```swiper.setAttribute('slides-per-view', '3');``` to update slidePerView property when screen size was changed. However I couldn't find anyway to do this on version 10. There is no method in documentation to implement this feature or am I wrong?!

### Suggested solution

```// swiper initialized```
```swiper.updateSlidesPerView(3);```

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this feature?

- [ ] I'm willing to open a PR

**Comment by kirgans** [comment, marked as answer]
[RTFM](https://swiperjs.com/swiper-api#param-breakpoints) 😁

**Reply by masarbazi** [reply]
Thanks. I was just looking at methods section, didn't realize this 😅
